docs/api/cassandra/cqlengine/models.rst

```diff
@@ -82,7 +82,7 @@ Model
 
     .. automethod:: create
 
-    .. method:: if_not_exists()
+    .. method:: if_not_exists(boolean)
```

Comment:
I see. Sorry for the confusion. I think we should try to get away from that inconsistency. Maybe we can default that parameter ``True`` in the setter and take it away later.